rocketpy/environment/environment.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=too-many-lines, broad-exception-caught, bare-except, raise-missing-from, consider-using-f-string, too-many-statements, too-many-instance-attributes, invalid-name, too-many-locals
```

Comment:
This seems to be the module with the most linting suggestions, just a reminder for the Evironment refactor PR that perhaps we can tackle some of them.

Reply:
Yes I'm aware of this.

We will tackle these problems in the other PR